docs/sanic/config.rst

```diff
@@ -12,9 +12,9 @@ Sanic holds the configuration in the `config` attribute of the application objec
 
     app = Sanic('myapp')
     app.config.DB_NAME = 'appdb'
-    app.config.DB_USER = 'appuser'
+    app.config['DB_USER'] = 'appuser'
```

Comment:
I think we could give example with dot notation and dict notation (we mention about it line above).